release/gff/gff.am-ethi.gff_amharic/gff.am-ethi.gff_amharic.model_info

```diff
@@ -0,0 +1,7 @@
+{
+    "license": "mit",
+    "languages": [
+        "am"

```

Comment:
In the directory path and .kps files, you used `am-Ethi` for the language tag.
(Whichever tag you settle on will also have to match the language tag in your Amharic keyboard)

Reply:
I realized Developer probably filled out `am-Ethi` for your project and made the directory.

gff_amharic uses `am`, so that's probably the right one to use for all the paths.